exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/metadata/MetadataVersion.java

```diff
@@ -145,6 +145,11 @@ public int compareTo(MetadataVersion o) {
      */
     public static final String V3_3 = "3.3";
 
+  /**
+   *  Version 4: Split the metadata cache file into summary and file metadata
+   */
+  public static final String V4 = "4";
```

Comment:
It occurs to me that for consistency we should follow a convention of `MajorVersion.MinorVersion`  i.e 4.0 instead of just 4.  In Version 3 we started adding the minor versions later but now that we are following that model, it would make sense to have a 2 digit scheme.  Let's discuss..

Reply:
Changed to 4.0